tests/integration/crunchers/test_cruncher_lead_gas.py

```diff
@@ -15,10 +15,10 @@ class TestDatabaseCruncherLeadGas(_DataBaseCruncherTester):
     tclass = DatabaseCruncherLeadGas
     tdb = pd.DataFrame(
         [
-            _msa + ["World", "Emissions|HFC|C5F12", "kt C5F12/yr", np.nan, 3.14],
-            _msa + ["World", "Emissions|HFC|C2F6", "kt C2F6/yr", 1.2, 1.5],
+            _msa + ["World", "Emissions|HFC|C5F12", "kt C5F12/yr", "", np.nan, 3.14],
```

Comment:
Is this empty in data that you download from SR1.5 database? If no, I'd be tempted to fill with an actual value even if it's just 'test'

Reply:
Yes, extra columns are often empty